eldritch/test_mythos.py

```diff
@@ -1470,6 +1472,101 @@ def setUp(self):
       monster.idx = idx
       monster.place = self.state.monster_cup
 
+  def testActivateRumor13(self):
+    self.state.mythos.append(Mythos13())
+    self.state.event_stack.append(Mythos(self.char))
+    self.resolve_until_done()
+    self.assertTrue(self.state.rumor)
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertFalse(self.state.rumor.failed)
+    self.assertNotIn(self.state.rumor, self.state.mythos)
+
+  def testRumor13Progress(self):
+    self.state.allies.extend(assets.CreateAllies()[:7])
+    self.state.event_stack.append(IncreaseTerror(6))
+    self.resolve_until_done()
+    self.state.mythos.append(Mythos13())
+    self.state.event_stack.append(Mythos(self.char))
+    self.resolve_until_done()
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertEqual(self.state.terror, 6)
+    self.assertEqual(len(self.state.boxed_allies), 6)
+
+    self.advance_turn(self.state.turn_number+1, "mythos")
```

Comment:
What mythos card gets activated the next turn?

Reply:
EventTest.add_turn().  Added a todo about edge cases.